apps/www/src/components/layout/header/index.tsx

```diff
@@ -69,12 +80,24 @@ export const PageHeader = async () => {
 
   return (
     <div
+      ref={headerRef}
       className={css({
         bg: 'pageBackground',
         borderBottomWidth: 1,
         borderBottomStyle: 'solid',
         borderBottomColor: 'divider',
+        position: 'sticky',
+        top: 0,
+        transition: 'transform 0.3s ease-out',
+        zIndex: 100,
       })}
+      style={{
+        transform: `translateY(${
+          scrollDirection === 'down'
+            ? -((headerRef.current?.clientHeight || 0) + 1)
```

Comment:
I believe this is fine in practice but reading from the DOM in render is generally a no-no?

Reply:
Don't really have many other options. Either that or the useEffect thing that was there previously.

Reply:
Do we need to know the actual height of the element? We can use something like max header height, which shouldn't make much of a difference even when the actual height is smaller.

Reply:
Just remembered, this can be solved with CSS: we have the value in `var(--sizes-header-height)` (which is responsive)

Reply:
Ah, nevermind, that's without the navigation bar … but it could be solved by adding a totalHeight to sizes.header in the Panda preset